vendor/Sharpmake/Sharpmake.Application.exe.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <runtime>
+        <gcServer enabled="true"/>
```

Comment:
What does it do?

Reply:
Generated when compiling Sharpmake. Google says: Specifies whether the common language runtime runs server garbage collection.